### ISSUE TITLE
Filter on duplicates by linked URLs

### ISSUE THREAD

**Issue by Matthias84** [FreshRSS/FreshRSS#2104]
In addition to the filter request, I would love to get an feature, that help to reduce the amount of items. It should merge different feeds by eliminating items that contain the same links to external third party pages.

For example: You subscribe two feeds of newspaper A and B. Within feed A is a article item, that text links to reference 1. Also at feed B is a (diffrent) article, that links to reference 1, too. The feature now marks feed B article as read, to avoid the dublicate.

This feature might become complex due to blacklisting URLs (think about embedded HTML widgets, ...), suppression priority, ...

**Comment by Alkarex** [comment]
Indeed, handling duplicates - either on the same feed or across several feeds - is a feature that is already quite high on my priority list.
See the related issue https://github.com/FreshRSS/FreshRSS/issues/948

We need to clarify what the behaviour should be in more details, in particular which article to keep unread.
Another open question is whether there should be an option doing some magic, or whether it should be based on explicit rules written by the user (e.g. mark articles in feed 3 as read when identical articles are found in feed 4 or feed 5).

**Comment by Matthias84** [comment]
I think we have the following options to solve this decisions
* priority lists for feeds
* interactive solution
* explicit definition of supression rules (mix which channels, which will win)
